tests/remote_attestation/maa/sgxlkl_enclave/oe_public/bits/types.h

```diff
@@ -0,0 +1,180 @@
+/**
+ * @file types.h
```

Comment:
This file should go away if we expose the cert-related APIs through OE host verify.

Reply:
There is no action item for now. if this happens we can submit a follow up PR. Let's don't block the PR with uncertainity for now.